### DISCUSSION TITLE
Add social platforms icons along with framework icons

### DISCUSSION THREAD

**Discussion by Rajat8699** [tailwindlabs/heroicons#1250] [Ideas]
Add social platforms icons like Telegram, facebook, etc. along with framework icons like ReactJs, NextJs, VueJs, etc.

**Comment by caleb8315** [comment]
You should try out https://github.com/onlook-dev/onlook

It will make developing in front end so much easier for your react applications. And they have an Ai feature that updates your code in real time! It's actually amazing!
